strings.xml

bolaram opened the pull request:
Some spellings updated :)

alicoding commented:
Hi @bolaram Thank you so much for the PR. I'm going to merge this PR, but I would like to ask you to also update the string on Transifex here https://www.transifex.com/webmaker/webmaker/webmaker-stringsxml/ so that it won't override what's in here if someone is going to change them on Transifex.

bolaram commented:
@alicoding thanks. i updated string on  https://www.transifex.com/webmaker/webmaker/webmaker-stringsxml/ .. i just added string suggestion there :) will you check that ?
